common/CMakeLists.txt

```diff
@@ -138,3 +138,15 @@ target_include_directories(identity_parser_test PRIVATE ${CMAKE_SOURCE_DIR})
 gtest_discover_tests(identity_parser_test)
 add_dependencies(check identity_parser_test)
 
+
+find_package(Boost 1.54 REQUIRED COMPONENTS filesystem)
```

Comment:
Filesystem is ways old...

I found it all the way back in `1.30`

Reply:
But you're sure it works with `1.54`?

Reply:
I thought that was what our CI was (used to be?) using.

Reply:
No, we're on like `1.7x` something